mythril/laser/ethereum/symbol_manager.py

```diff
@@ -0,0 +1,180 @@
+from z3 import BitVec, BitVecRef, is_bv
+
+
+class Sym:
```

Comment:
What do you think about renaming this to Symbol?

Reply:
It's fine. This code is picked from my SMT abstraction work where I chose 'Sym' to just avoid name conflict with pySMT which happens to provide a function 'Symbol'. Anyway, the conflict can be avoided by using a full name of pySMT Symbol in the future.